_pages/story.md

```diff
@@ -0,0 +1,70 @@
+
+A challenge is that each agency has its own collection of materials related to its regulations - with some types of documents in common with other agencies and some unique to one agency. Some agencies use different names for similar types of documents.
+
+A few examples:
+
+* [CFPB](http://www.consumerfinance.gov/regulations/) includes "official interpretations" in its regulations (as an appendix to the main part of the regulation, all published together in the Code of Federal Regulations). CFPB also includes "section-by-section analysis" in its Federal Register notices. CFPB's eRegulations displays both of those types of material to help readers understand regulations.
+* [ATF](https://www.atf.gov/rules-and-regulations) publishes "rulings" and "open letters" that clarify aspects of its regulations. It also publishes plain-language Q&As, newsletters, and guidebooks to help explain its regulations. ATF's eRegulations does not integrate those types of material, but it could in the future.
+* [FEC](http://www.fec.gov/law/law.shtml) publishes "advisory opinions" and "Matters Under Review" that clarify aspects of its regulations, among other kinds of material.
+
+## How it works
+
+[Code and documentation](technology/) explains this in more detail, but in short: the parser eats XML from many different sources and writes that to an API. The UI then reads from that API.
+
+There is currently one UI for eRegulations, but the intent is for the parser to be usable outside of this one interface into regulations, and for the API to be usable outside of this one interface.
+
+For example, people could build a regulations authoring tool with a different UI. Some regulatory agencies have specific needs that may require a different UI.
+
+The current codebase is somewhat tangled together, since there is just one UI right now, but supporting flexibility is an important ongoing goal.
+
+## Advice for ongoing cross-agency development
+
+Everyone is free to use and adapt eRegulations as they wish. We encourage you to build your adaptation in the open (and with a public domain + CC0 license), and to build it in close coordination with other contributors, so everyone can benefit from each other's work.
+
+### Working with the shared codebase and building agency-specific features
+
+The procedure if you develop a feature that would be more platform-oriented: **regulations-core**, **regulations-site**, and **regulations-parser** are the core platform; they're shared code. We don't have a very good "raw" eRegulations right now, though we're slowly moving there. "Raw" eRegulations is basically CFPB eRegulations right now.
+
+At this point, you would probably never run **regulations-site** or **regulations-core** on their own. Instead, they're libraries - you can modify them as libraries. Effectively, you point the wrapper to local checkouts.
+
+In general we are not extending the core repo further when we make changes to an agency-specific setup.
```

Comment:
This might need to be expanded a bit. I think it's supposed to be something like "Adding agency-specific styles, markup, and even certain types of features should not require modifications to the core repos".